lib/fun_with_flags/store/persistent/in_memory.ex

```diff
@@ -0,0 +1,163 @@
+defmodule FunWithFlags.Store.Persistent.InMemory do
+  alias FunWithFlags.{Config, Gate, Flag, Store.Persistent.Ecto.Record}
+
+  def worker_spec do
+    import Supervisor.Spec, only: [worker: 3]
+
+    worker(__MODULE__, [], restart: :permanent)
+  end
+
+  def child_spec(opts \\ []) do
+    %{
+      id: __MODULE__,
+      start: {__MODULE__, :start_link, [opts]}
+    }
+  end
```

Comment:
I'm wondering why you need both `worker_spec` and `child_spec`. Some comments would be helpful!

Reply:
the worker_spec is probably a left over from when I was first doing this (copying from the Ecto Adapter), which I forgot to remove